src/features/werkbericht/WerkBericht.vue

```diff
@@ -1,26 +1,39 @@
 <template>
-  <article :class="read && 'read'">
+  <article :class="{ read: read }">
     <small v-if="showType && bericht.types.length">
       {{ bericht.types.join(", ") }}
     </small>
 
     <div class="heading-container">
+      <div class="heading-top-row">
+        <time :datetime="bericht.date.toISOString()">{{
+          localeString(bericht.date)
+        }}</time>
+
+        <menu>
+          <input
```

Comment:
I had some doubts about screen reader support for an input with only a title attribute, [but it actually seems widely supported](https://www.powermapper.com/tests/screen-readers/labelling/input-text-title/). Good to know!